src/pages/welcome.tsx

```diff
@@ -103,6 +118,19 @@ export const WelcomePage: React.FC = () => {
             {t('welcomePage.button')}
           </GetStartedButton>
         </div>
+        <Subtext>
+          {previousRunClasses && (
+            <p>
+              {!previousRunIsValid ? (
```

Comment:
tiny stylistic thing, but would find it clearer if we swapped the ternary so it wasn't based on a negation
```
previousRunIsValid ? <Link ... /> : t('welcomePage.previousRunExpired')
```